core/src/main/java/org/hibernate/ogm/storedprocedure/ProcedureQueryParameters.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.storedprocedure;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Named or positional parameters to pass to a stored procedure.
+ *
+ * @author Davide D'Alto
+ */
+public class ProcedureQueryParameters {
```

Comment:
So I have a question similar to what Sergey asked but a bit different: do we need a class specific to OGM? Can't we reuse ORM's mechanisms? (I didn't check so it's a real question)

A big part of my work on the ORM 5.2 port was to try to reduce to the minimum the number of classes reimplemented in OGM so I'm wondering if maybe we could be a bit more "ORM integrated"?

I have the same interrogation for a few other classes.

I haven't digged if it was possible or not so this question is more to share a concern I have. Might be totally misplaced.

Reply:
This PR was sent when we were still on ORM 5.1

I'm pretty sure at the time I couldn't reuse existing classes. I'm rebasing this PR now.
I will check what I can remove.

Reply:
Yeah, as discussed with Sergey on HipChat, not sure we can do better, just wondering :).